app/models/stat/anime_genre_breakdown.rb

```diff
@@ -1,52 +1,15 @@
 class Stat
   class AnimeGenreBreakdown < Stat
-    # fully regenerate data
-    def recalculate!
-      genres = user.library_entries.eager_load(anime: :genres)
-                   .where.not(genres: { slug: nil })
-                   .group(:'genres.slug').count
+    include Stat::GenreBreakdown
 
-      self.stats_data = genres
-      stats_data['total'] = genres.values.reduce(:+)
-
-      save!
-    end
-
-    def self.increment(user, library_entry)
-      record = user.stats.find_or_initialize_by(
-        type: 'Stat::AnimeGenreBreakdown'
-      )
-      # set default to total if it doesn't exist
-      record.stats_data['total'] = 0 if record.new_record?
-      record.increment(library_entry)
+    # for recalculate!
+    def media_column
+      :anime
     end
 
-    def increment(library_entry)
-      library_entry.media.genres.each do |genre|
-        stats_data[genre.slug] = 0 unless stats_data[genre.slug]
-
-        stats_data[genre.slug] += 1
-        stats_data['total'] += 1
-      end
-
-      save!
-    end
-
-    def self.decrement(user, library_entry)
-      record = user.stats.find_by(type: 'Stat::AnimeGenreBreakdown')
-      return unless record
-      record.decrement(library_entry)
-    end
-
-    def decrement(library_entry)
-      library_entry.media.genres.each do |genre|
-        next unless stats_data[genre.slug]
-
-        stats_data[genre.slug] -= 1
-        stats_data['total'] -= 1
-      end
-
-      save!
+    # for class methods
+    def self.media_column
+      'Anime'
```

Comment:
But... this isn't a column... why did you name this method `media_column`?  The method name only makes sense for `def media_column` because it's returning the name of the media kind...